crypto/cosi.go

```diff
@@ -316,6 +316,25 @@ func (m *Mask) SetBit(i int, enable bool) error {
 	return nil
 }
 
+// GetPubKeyFromMask will return pubkeys which masked either zero or one depending on the flag
```

Comment:
not clear on the purpose of this function. please elaborate a bit more on how to use it.

Reply:
This is to show which validators had signed in the cosi message when the leader received their cosign message. I used it to show the new nodes are active in the consensus process

Reply:
I add more comments on it